.github/workflows/ci.yml

```diff
@@ -0,0 +1,197 @@
+name: CI
+
+on:
+  push:
+    tags:
+      - '*'
+    branches:
+      - master
+      - develop
+  pull_request:
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up JDK 11
+        uses: actions/setup-java@v1
+        with:
+          java-version: 11
+      - name: Run tests
+        run: sbt coverage test coverageReport
+      - name: Aggregate coverage data
+        if: ${{ always() }}
+        run: sbt coverageAggregate
+      - name: Submit coveralls data
+        if: ${{ always() }}
+        run: sbt coveralls
+        env:
+          COVERALLS_REPO_TOKEN: ${{ secrets.COVERALLS_REPO_TOKEN }}
+
+  publish_batch_transformer:
+    needs: test
+    if: startsWith(github.ref, 'refs/tags/')
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Github
+        uses: actions/checkout@v2
+      - uses: coursier/cache-action@v6
+      - name: Set up Python to 2.x
+        uses: actions/setup-python@v1
+        with:
+          python-version: '2.x'
```

Comment:
I don't like depending on an unsupported version of python.  But let's fix this another time, not now.